x-pack/plugins/security_solution/public/timelines/components/flyout/bottom_bar/index.tsx

```diff
@@ -14,25 +14,32 @@ import { DataProvider } from '../../timeline/data_providers/data_provider';
 import { flattenIntoAndGroups } from '../../timeline/data_providers/helpers';
 import { DataProviders } from '../../timeline/data_providers';
 import { FlyoutHeaderPanel } from '../header';
+import { TimelineTabs } from '../../../store/timeline/model';
 
 export const FLYOUT_BUTTON_CLASS_NAME = 'timeline-flyout-button';
 
 export const getBadgeCount = (dataProviders: DataProvider[]): number =>
   flattenIntoAndGroups(dataProviders).reduce((total, group) => total + group.length, 0);
 
-const SHOW_HIDE_TRANSLATE_X = 50; // px
+const SHOW_HIDE_GLOBAL_TRANSLATE_X = 50; // px
```

Comment:
nit: The OG code should have been named `SHOW_HIDE_GLOBAL_TRANSLATE_Y`. Not your mistake, but would you be willing to change it?